spec/models/ext_management_system_spec.rb

```diff
@@ -804,4 +804,12 @@ def deliver_queue_message(queue_message = MiqQueue.order(:id).first)
       expect(ems.supports_create_security_group).to be(true)
     end
   end
+
+  describe "#image_name", :providers_common => true do
+    described_class.concrete_subclasses.each do |klass|
+      it "has a vendor asset for #{klass.ems_type}" do
+        expect(ActionController::Base.helpers.asset_path("svg/vendor-#{klass.new.image_name}.svg")).to_not be_blank
```

Comment:
Interesting I wonder if image_name could be a class method instead?  It seems like it uses `#emstype.downcase` which is a virtual_column for `self.class.ems_type`

Reply:
I actually started down that path and a bunch of providers have overridden the image_name instance method. I figured I'd rather get this spec in first before cleaning up anything.